client/my-sites/sharing/connections/connections.jsx

```diff
@@ -11,33 +11,28 @@ import { localize } from 'i18n-calypso';
 import QueryKeyringServices from 'components/data/query-keyring-services';
 import SharingServicesGroup from './services-group';
 
-class SharingConnections extends Component {
-	static propTypes = {
-		connections: PropTypes.object,
-		translate: PropTypes.func,
-	};
+const SharingConnections = ( { connections, translate } ) => (
+	<div className="sharing-settings sharing-connections">
+		<QueryKeyringServices />
+		<SharingServicesGroup
+			type="publicize"
+			title={ translate( 'Publicize Your Posts' ) }
+			connections={ connections } />
+		<SharingServicesGroup
+			type="other"
+			title={ translate( 'Other Connections' ) }
+			connections={ connections } />
+	</div>
+);
 
-	static defaultProps = {
-		connections: Object.freeze( {} ),
```

Comment:
Introduced in dc97c60f1465bf56222c0fd61710a35179cefd1d by yours truly, at a time when I assumed that was convention for objects and arrays in default props.